README.md

```diff
@@ -306,6 +307,26 @@ In order to achieve a Zero Copy message delivery, applications need to both enab
     </profiles>
     ```
 
+### Large data transfer over lossy network
+
+In default, Fast DDS uses UDPv4 for the data communication over the network.
+UDP has its own merit for realtime capability, and many applications rely on UDP.
+But this depends on the application requirement, if it requires a more reliable network communication with large data samples over the lossy network, probably TCP flow control is more suitable.
+
+Fast DDS provides the configuration to modify the builtin transports via environmental variable `FASTDDS_BUILTIN_TRANSPORTS` as below.
```

Comment:
```suggestion
Because of this reason, Fast DDS provides the possibility to modify its builtin transports via an environmental variable `FASTDDS_BUILTIN_TRANSPORTS`, allow for easily changing the transport layer to TCP when needed:
```